src/main/java/core/basesyntax/FileReaderImpl.java

```diff
@@ -0,0 +1,17 @@
+package core.basesyntax;
+
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.List;
+
+public class FileReaderImpl implements FileReader{
+    @Override
+    public List<String> read(String file) {
```

Comment:
The method signature `read` could be more descriptive. Consider renaming it to something like `readLinesFromFile` to make it clear that it reads the lines from a file.